daemons/controld/controld_cib.c

```diff
@@ -153,6 +153,12 @@ do_cib_replaced(const char *event, xmlNode * msg)
         return;
     }
 
+    if (pcmk_is_set(controld_globals.fsa_input_register, R_SHUTDOWN)) {
+        crm_debug("Ignoring CIB replace notification because we're shutting "
+                  "down");
+        return;
+    }
+
```

Comment:
I don't think we can do this. R_SHUTDOWN is set when the shutdown request is sent to the DC; the node still has to behave normally for a while after that as resources are stopped.